app/Filament/Pages/BrandingBar.php

```diff
@@ -0,0 +1,321 @@
+
+        return $record;
+    }
+
+    protected function getSavedNotification(): ?Notification
+    {
+        $title = $this->getSavedNotificationTitle();
+
+        if (blank($title)) {
+            return null;
+        }
+
+        return Notification::make()
+            ->success()
+            ->title($this->getSavedNotificationTitle());
+    }
+
+    protected function getSavedNotificationTitle(): ?string
+    {
+        return __('filament-panels::pages/auth/edit-profile.notifications.saved.title');
+    }
+
+    protected function getRedirectUrl(): ?string
+    {
+        return null;
+    }
+
+    protected function getNameFormComponent(): Component
+    {
+        return TextInput::make('name')
+            ->label(__('filament-panels::pages/auth/edit-profile.form.name.label'))
+            ->required()
+            ->maxLength(255)
+            ->autofocus();
+    }
+
+    protected function getEmailFormComponent(): Component
+    {
+        return TextInput::make('email')
+            ->label(__('filament-panels::pages/auth/edit-profile.form.email.label'))
+            ->email()
+            ->required()
+            ->maxLength(255)
+            ->unique(ignoreRecord: true)
+            ->hint(fn (Get $get): string => $get('is_email_visible_on_profile') ? 'Visible on profile' : 'Not visible on profile');
+    }
+
+    protected function getPasswordFormComponent(): Component
+    {
+        return TextInput::make('password')
+            ->label(__('filament-panels::pages/auth/edit-profile.form.password.label'))
+            ->password()
+            ->rule(Password::default())
+            ->autocomplete('new-password')
+            ->dehydrated(fn ($state): bool => filled($state))
+            ->dehydrateStateUsing(fn ($state): string => Hash::make($state))
+            ->live(debounce: 500)
+            ->same('passwordConfirmation');
+    }
+
+    protected function getPasswordConfirmationFormComponent(): Component
+    {
+        return TextInput::make('passwordConfirmation')
+            ->label(__('filament-panels::pages/auth/edit-profile.form.password_confirmation.label'))
+            ->password()
+            ->required()
+            ->visible(fn (Get $get): bool => filled($get('password')))
+            ->dehydrated(false);
+    }
+
+    /**
+     * @return array<int | string, string | Form>
+     */
+    protected function getForms(): array
+    {
+        return [
+            'form' => $this->form(
+                $this->makeForm()
+                    ->schema([
+                        $this->getNameFormComponent(),
+                        $this->getEmailFormComponent(),
+                        $this->getPasswordFormComponent(),
+                        $this->getPasswordConfirmationFormComponent(),
+                    ])
+                    ->operation('edit')
+                    ->model($this->getUser())
+                    ->statePath('data'),
+            ),
+        ];
+    }
```

Comment:
I don't think here (or in any of the other profile page files OTHER than the one with these fields, the ProfileInformation one, these are needed) Since these forms fields are not on those pages.